README.md

```diff
@@ -93,9 +107,41 @@ and was compiled on:
   * Mac OS X ([see how](/doc/CompilationMac.md))
   * Windows ([see how](/doc/CompilationWindows.md) - partially supported)
 
+### Docker images
+
+Run the following commands to pull and run libnabo in a docker container
+
+```shell
+docker pull norlabulaval/libpointmatcher:latest-ubuntu-focal
+
+docker run -it --rm norlabulaval/libpointmatcher:latest-ubuntu-focal
+```
+
+See
+available [libpointmatcher image tags](https://hub.docker.com/repository/docker/norlabulaval/libpointmatcher/)
```

Comment:
The link page currently does not exist on DockerHub

Reply:
Yes the CI release build needed to finish its job. The image are on dockerhub as of now